stacks-signer/src/v0/signer.rs

```diff
@@ -448,10 +482,7 @@ impl Signer {
             self.test_reject_block_proposal(block_proposal, &mut block_info, block_response);
 
         if let Some(block_response) = block_response {
-            // We know proposal is invalid. Send rejection message, do not do further validation
-            if let Err(e) = block_info.mark_locally_rejected() {
```

Comment:
I'm not clear on why this is no longer needed. Can you explain that part?

Reply:
We do not store the block in the database. Why bother mutating it to never store it?

Reply:
Ah, right. Thanks!